build-a-react-native-application-for-ios-and-android-from-start-to-finish/notes/01-create-a-react-native-app-and-run-it-on-the-ios-simulator-and-android-emulator.md

```diff
@@ -0,0 +1,87 @@
+# Create a React Native App and Run it on the iOS Simulator and Android Emulator
+
+**[📹 Video](https://egghead.io/lessons/react-native-create-a-react-native-app-and-run-it-on-the-ios-simulator-and-android-emulator)**
+
+
+## Installing dependencies
+
+### Node.js
+Node.js is required to install the `react-native` package and any additional packages your project requires. Head over to [Node.js download](https://nodejs.org/en/download/), download installer for OS and follow installation instructions.
+
+### Visual Studio Code
+Any text editor can be used - Sublime Text, Atom, Vim etc - but if you do not yet have a text editor of choice, check out [Visual Studio Code](https://code.visualstudio.com/download).
+
+### React Native
+Navigate to [react native docs](https://reactnative.dev/docs/environment-setup), select `React Native CLI Quickstart` tab, and choose your dev OS and target OS - e.g. macOS and iOS.
+
+> Note: iOS development can only be done on a macOS device, due to Apple licensing. If you are planning to build for iOS make sure you have a macOS dev device, otherwise follow the Android instructions to get setup.
+
+The docs will step you through each dependency you need to setup, before installing the `react native` package. Go through the `Installing dependencies` section for each target OS, even if you are only planning to build for one. This will save a lot of time in the future 🙂
```

Comment:
```suggestion
👍 The docs will step you through each dependency you need to setup, before installing the `react native` package. Go through the `Installing dependencies` section for each target OS, even if you are only planning to build for one. This will save a lot of time in the future 🙂
```